setup.cfg

```diff
@@ -68,10 +68,11 @@ install_requires =
     numpy==1.19.0; python_version=='3.6' and platform_machine!='loongarch64' and platform_python_implementation=='PyPy'
     numpy==1.20.0; python_version=='3.7' and platform_machine!='loongarch64' and platform_python_implementation=='PyPy'
     numpy==1.22.2; python_version=='3.8' and platform_machine!='loongarch64' and platform_python_implementation=='PyPy'
+    numpy==1.25.0; python_version=='3.9' and platform_machine!='loongarch64' and platform_python_implementation=='PyPy'
```

Comment:
This depends a bit on the definition of "oldest supported". If it is "has a wheel", then this is true. In terms of a more broadly-defined support, pypy 3.8 & 3.9 have been passing the full numpy test suite (in conda-forge) as far back as [1.19](https://github.com/conda-forge/numpy-feedstock/pull/196#issuecomment-1097440489)

Reply:
OTOH, since there are no loongarch64 wheels either, I think we could actually stop distinguishing CPython & PyPy after a certain cut-off (say 3.8, as compatibility has improved quite a bit since 3.6/3.7)

Reply:
> This depends a bit on the definition of "oldest supported". If it is "has a wheel", then this is true

Yep, that's what we've always used.

>  In terms of a more broadly-defined support, pypy 3.8 & 3.9 have been passing the full numpy test suite

I don't think it's practically useful to start targeting older versions, both because  they may not actually work reliably and (more importantly) because it then requires building `numpy` from source, which many `pip` & co users are not set up for.